patches/extensions-browser-extension_event_histogram_value.h.patch

```diff
@@ -18,6 +18,7 @@ index d3d964de413cc744fbabe87eb65fafc2cb1ddfd0..8f216d22b1223d7477f8b8523fd3a4d1
 +  BRAVE_REWARDS_ALL_NOTIFICATIONS_DELETED,
 +  BRAVE_REWARDS_GET_NOTIFICATION,
 +  BRAVE_REWARDS_GET_ALL_NOTIFICATIONS,
++  BRAVE_WALLET_FAILED,
```

Comment:
we either need to just use one value for all of them `BRAVE` or find another way to add these. I don't think there's really much benefit to adding them at this point